src/plugins/expression/expression-tile.tsx

```diff
@@ -30,12 +30,11 @@ export const ExpressionToolComponent: React.FC<ITileProps> = observer((props) =>
     model, readOnly, documentContent, tileElt, scale } = props;
   const content = model.content as ExpressionContentModelType;
   const mf = useRef<MathfieldElement>(null);
+  const readOnlyMf = readOnly ? useRef<MathfieldElement>(null) : null;
   const trackedCursorPos = useRef<number>(0);
   const ui = useUIStore();
 
-  if(mf.current && ui) {
-    mf.current.addEventListener("focus", () => ui.setSelectedTileId(model.id));
-  }
+  if(mf.current && ui) mf.current.addEventListener("focus", () => ui.setSelectedTileId(model.id));
```

Comment:
This should be in a useEffect so that it will be run after mf.current has been set.
Also note that currently on each render a new event listener is going to be added. You probably only want to do this one time.